{"version":3,"sources":["../../../source/library/error/worker-client-internal-error.js"],"names":["WorkerClientRejectedError","WorkerClientInternalError","constructor","internalError","message","_internalError"],"mappings":"AAAA,SAASA,yBAAT,QAA0C,mCAA1C;;AAEA,MAAMC,yBAAN,SAAwCD,yBAAxC,CAAkE;;AAEhEE,EAAAA,WAAW,CAACC,aAAD,EAAgB;AACzB,UAAO,sDAAqDA,aAAa,CAACC,OAAQ,IAAlF;AACA,SAAKC,cAAL,GAAsBF,aAAtB;AACD;;AAEgB,MAAbA,aAAa,GAAG;AAClB,WAAO,KAAKE,cAAZ;AACD,GAT+D;;;;AAalE,SAASJ,yBAAT","sourcesContent":["import { WorkerClientRejectedError } from './worker-client-rejected-error.js'\n\nclass WorkerClientInternalError extends WorkerClientRejectedError {\n\n  constructor(internalError) {\n    super(`The server exited with the internal error message '${internalError.message}'.`)\n    this._internalError = internalError\n  }\n\n  get internalError() {\n    return this._internalError\n  }\n\n}\n\nexport { WorkerClientInternalError }\n"],"file":"worker-client-internal-error.js"}